srcpkgs/musl-legacy-compat/template

```diff
@@ -1,15 +1,18 @@
 # Template file for 'musl-legacy-compat'
 pkgname=musl-legacy-compat
 version=0.3
-revision=1
-noarch=yes
+revision=2
 bootstrap=yes
-only_for_archs="i686-musl x86_64-musl armv5tel-musl armv6l-musl armv7l-musl aarch64-musl mips-musl mipshf-musl mipsel-musl mipselhf-musl"
 short_desc="Legacy compatibility headers for the musl libc"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="BSD"
 homepage="http://www.voidlinux.eu"
 
+case "${XBPS_TARGET_MACHINE"} in
```

Comment:
Bracket is outside of quotes, i'd remove the brackets altogether since that is what most do (419 vs 37)